### DISCUSSION TITLE
Trying to redefine a prop set in another module produces a build error

### DISCUSSION THREAD

**Discussion by jstansbe** [vercel/next.js#14629] [Help]
# Bug report

## Describe the bug

**Trying to redefine a prop set in another module produces a build error.**
Cannot redefine property: `REACT_NAVIGATION_REDUX_DEVTOOLS_EXTENSION_INTEGRATION_ENABLED`

## To Reproduce

Have/include react-navigation in your project

## Expected behavior

Page would build without an error

## Screenshots

![image](https://user-images.githubusercontent.com/1250522/85925562-00498700-b84e-11ea-9378-f68c7aa04143.png)

## System information

"next": "^9.4.4",
"@react-navigation/native": "^5.6.1",

## Other
Text version of screenshot

Server Error
TypeError: Cannot redefine property: REACT_NAVIGATION_REDUX_DEVTOOLS_EXTENSION_INTEGRATION_ENABLED

This error happened while generating the page. Any console logs will be displayed in the terminal window.
Source
Object../node_modules/@react-navigation/core/lib/commonjs/BaseNavigationContainer.js
webpack:///BaseNavigationContainer.tsx (37:0)
__webpack_require__
webpack:///webpack/bootstrap (21:0)
Object../node_modules/@react-navigation/core/lib/commonjs/index.js
webpack:///index.tsx (3:0)
__webpack_require__
webpack:///webpack/bootstrap (21:0)
Object../node_modules/@react-navigation/native/lib/commonjs/index.js
webpack:///index.tsx (1:0)
__webpack_require__
webpack:///webpack/bootstrap (21:0)
__webpack_require__
webpack:///webpack/bootstrap (21:0)
__webpack_require__
webpack:///webpack/bootstrap (21:0)

Call Stack
__webpack_require__
webpack:///webpack/bootstrap (21:0)
__webpack_require__
webpack:///webpack/bootstrap (21:0)
__webpack_require__
webpack:///webpack/bootstrap (21:0)
<unknown>
webpack:///webpack/bootstrap (89:0)

**Comment by jamesmosier** [comment]
Are you able to share some example code? Where are you including the package? What other pages do you have?
